src/xmagics/execution.cpp

```diff
@@ -116,7 +127,7 @@ namespace xcpp
 
         try
         {
-            if (number == 0ul)
+            if (number == 0)
```

Comment:
you should keep `0ul` (or `std::size_t(0)`) to avoid annoying warnings when compiling.

Reply:
Well actually I couldn't get the arguments as size_t with argparse (still thinking of a way to do that) so `number`, `repeat` and `precision` are not size_t anymore...

Reply:
Ah indeed I missed the change sizeèt => int